contents/articles/2016-07_pupil-service/index.md

```diff
@@ -0,0 +1,10 @@
+---
+ title: Pupil Service
+ date: Tue Jul 07 2016 09:47:48 GMT+0700 (ICT)
+ author: Pupil Dev Team
+ subtitle: Pupil Service is a new app in release v0.8.0...
+ ---
+
+Pupil Service is a new app in release v0.8.0
+
+We have added a third app to the Pupil project: Pupil Service. It is like Pupil Capture except it does not have a world video feed or GUI. It is intended for AR and VR eye tracking setups. Pupil Service is meant to run in the background and controlled via network commands only. The service process has no GUI. The tools introduced in the HMD-eyes project are made to work with Pupil Service and Pupil Capture alike.
```

Comment:
Add paragraph: 
For more details head over to the release notes and download Pupil v0.8.0 here.

`here` is the anchor.